pygmsh/surface_base.py

```diff
@@ -4,11 +4,13 @@
 
 class SurfaceBase(object):
     _ID = 0
+    num_edges = 0
 
-    def __init__(self, id0=None):
+    def __init__(self, num_edges=0, id0=None):
```

Comment:
In light of backwards compatibility, the order of the arguments should probably be reversed.